docs/05-technical-reference/00-custom-resources/README.md

```diff
@@ -6,7 +6,7 @@ A custom resource (CR) is an extension to the Kubernetes API which allows you to
 
 | Area | Custom resource |
 | ---- | -------------- |
-| Application Connectivity | [Application](ac-01-application.md), [CompassConnection](ra-01-compassconnection.md) |
```

Comment:
The Application Custom Resource has not been removed from Kyma repo (yet!!!) and is installed as part of compass-runtime-agent until it will be finally migrated to Application Connector Module. We should not remove its description.